src/Javis.jl

```diff
@@ -79,18 +123,78 @@ The most simple form of an action (if there are no `args`/`kwargs`) just calls
 Action(frames, func::Function, args...; kwargs...) = Action(frames, nothing, func, args...; kwargs...)
 
 """
```

Comment:
I think this docstring is good but since we are so heavily extending what an `Action` is, we might want to link to some bigger tutorial here about different invocations of all the different Actions we have made. That way, a user could now exactly what the differences are between each form. (ping #25 )

Reply:
In documenter they will be all underneath each other such that this is the first which holds no real information as the actual examples are given in the later `Actions` if that makes sense? There `kwargs` and `args` are explained. I also do link to `javis` for usage. I'm confused by what you want to add here I guess.

Reply:
Oh, it was more just a general remark. Resolving issue.